### ISSUE TITLE
18.3.0-rc3

### ISSUE THREAD

**Issue by jrha** [quattor/release#322]
Top level issue tracker for any blocking issues discovered with 18.3.0-rc3.

**Comment by jrha** [comment]
Examples are failing to build due to an (as yet) unidentified bug in the grid template library.

```
     [echo] Cluster: umd3 in /tmp/scdb-vanilla/cfg/clusters/grid
     [panc] 13/13 template(s) being processed
     [panc] validation error [/tmp/scdb-vanilla/cfg/clusters/grid/umd3/profiles/cream.example.org.pan]
     [panc] validation failed because exception was thrown
     [panc] element path: '/software/components/sudo'
     [panc] path '/software/components/sudo' bound to type element in [/tmp/scdb-vanilla/cfg/quattor/dev/components/sudo/validation.pan:?]
     [panc] caused by: evaluation error [/tmp/scdb-vanilla/cfg/quattor/dev/components/sudo/validation.pan:28.18-28.55]
     [panc] match() arguments must all be strings; list found
     [panc] >>> call stack trace 
     [panc] >>> [/tmp/scdb-vanilla/cfg/quattor/dev/components/sudo/validation.pan:28.18-28.55]
     [panc] >>> [/tmp/scdb-vanilla/cfg/quattor/dev/components/sudo/validation.pan:67.5-67.49]
     [panc] >>> [/tmp/scdb-vanilla/cfg/clusters/grid/umd3/profiles/cream.example.org.pan:73.37-73.92]
     [panc] >>> ====================
```

**Comment by jrha** [comment]
Bug fixed in quattor/configuration-modules-core#1264, new RC upcoming.
